### DISCUSSION TITLE
About community repositories

### DISCUSSION THREAD

**Discussion by patricklodder** [dogecoin/dogecoin#2430] [General]
I was triggered by the following comment in #2396

_Originally posted by @fmhc in https://github.com/dogecoin/dogecoin/issues/2396#issuecomment-890118370_

I think dogecoin should evolve and have a doge core repo for real core reference stuff, as @patricklodder said, and room for some supported / community approved repos in the dogecoin project domain. When will we have 20 signers for a release? We must make the dev community more integrative imho

-----

Currently, adding a repository to https://github.com/dogecoin automatically makes the people that are a member of the "core devs" team ultimately responsible for maintaining that repository. Over time, maintainers lose interest, as is the case for [dogecoin/dogecoin.com](https://github.com/dogecoin/dogecoin.com) for example, and then core dev team members get an additional job. That's not sustainable, and relies too much on a slow-to-change team. This was the foremost reason why I created the [rosetta-dogecoin "org"](https://github.com/rosetta-dogecoin) instead of adding it to the Dogecoin GitHub "org". The community is much broader than just this org.

If we change the terminology a bit, then we may be able to conceive another effort, that is also easier to manage in case that maintainers disappear: how about instead of trying to "approve" and imply some sort of authority from the org, we instead keep a list of things that people have reviewed. For example, I have reviewed [bitcoinjs-lib](https://github.com/bitcoinjs/bitcoinjs-lib) countless times for Dogecoin usage over the past 7 or so years, and I could totally do it again and log any findings. That way, we can keep a list of repositories that can be used to achieve specific goals and when it was last looked at by someone, what they looked at and what they found out, and/or what they use it for.

With this, the "org" here stays clean and acts as a source of information rather than the definitive authority and source for all things Dogecoin, which I feel suits a decentralized crypto much more than taking on an authoritative stance? Open for thoughts!

**Comment by Tattskemm** [comment]
Hi  my name is Kemju and I had been a member since last year yet due to lack of communication and restriction I was opt to do what I can to figure what was happening last year and where this access and leading to github and I was verify skeptical before where will I start and who will guide me, I was hoping if you would be kind enough to tell me or lead me were I should get started and how will I seek guidance or mentorship. Although I have been trying to learn this on my own not knowing basic and not talking to anyone will only lead me to nothing. I want to work with others as a team and I was wondering if you can give me some tips on how to check exsisting project I may signup with me not knowing the details and the doś and dont. I want to change that and learn how to abide rules. I sorry if it was a bit lenghty if its not too much to ask can you guide me please. Hoping to hear from you and hoping to make changes with everything that I started with a wrong foot. With utmost respect. Kemju

**Comment by AbcSxyZ** [comment]
In my opinion, all depends.

There are different tools more or less related to Dogecoin Core, some of them like libraries will have a direct impact on software utilization.
**Some stuff would make sense to be actively supported by the community and managed by a "known authority" like it is for Dogecoin Core, while listing some may be enough. I think we should do both.**

Maintenance issue are the same for Dogecoin Core, risks exists of having no more maintainers. There is a huge question about how to be able to increase the developer base and manage responsibilities to keep the repository maintained.
If we are able to attract more devs, build collaboration around a library for example and find invested volunteers, it may be possible to have multiple repos supported actively.

There is this question of incentive, find people who want and/or need to do that and being able to integrate them.
It may be even possible to financially support some work I guess, millions devoted to Dogecoin development are sleeping in someone's pocket ! They can do some good for the open source ecosystem around doge :)
